README.md

```diff
@@ -1 +1,130 @@
-# maryam0007.github.io
\ No newline at end of file
+# Portfolio-setup-and-mobile-first
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖<a name="about-project">Portfolio-Project</a>
+<p>This is a project which shows the portfolio setup and some mobile first changes</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Html</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Css</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- ** An html and css file reflecting the first page of portfolio project **
+- ** A gitignore file **
+- ** A file containing linters **
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Online version link](https://maryam0007.github.io/maryam007.github.io/)
```

Comment:
- [ ] Please I observed that the live demo link is broken; kindly check the address very well, I think there is an error. 👍 
```suggestion
[Online version link](https://maryam0007.github.io)
```